doc/rippled-example.cfg

```diff
@@ -606,30 +606,49 @@
 #   Partial pathnames will be considered relative to the location of
 #   the rippled.cfg file.
 #
-#   [node_db]       Settings for the NodeDB (required)
-#   [temp_db]       Settings for the look-aside temporary db (optional)
-#   [import_db]     Settings for performing a one-time import (optional)
+#   [node_db]               Settings for the Node Database (required)
 #
 #   Format (without spaces):
-#       One or more lines of key / value pairs:
+#       One or more lines of case-insensitive key / value pairs:
 #       <key> '=' <value>
 #       ...
 #
-#   Examples:
-#       type=rocksdb
-#       path=db/rocksdb
+#   Example:
+#       type=nudb
+#       path=db/nudb
+#
+#   The "type" field must be present and controls the choice of backend:
+#
+#   type = NuDB
+#
+#       NuDB is high performance, but only works with Solid State (SSD)
```

Comment:
I'd say a little here about what NuDB is.  "NuDB is a high-performance database written by Ripple Labs and optimized for rippled's use cases." or something like that.